src/components/storagePools/storagePoolOverviewTab.jsx

```diff
@@ -23,8 +23,6 @@ import { DescriptionList, DescriptionListGroup, DescriptionListTerm, Description
 import { storagePoolId } from '../../helpers.js';
 import cockpit from 'cockpit';
 
-import 'form-layout.scss';
```

Comment:
This is a bit unclear and buggy in the current code -- a lot of files (such as this one) include the CSS, but the only place where it is actually being used is src/components/vm/overview/bootOrder.jsx . That one would need to be ported first.

Reply:
bootOrder.jsx is now ported in 1d680c5833cbd7bcf57d26cd67bff3cc3a976a26